src/vscode/src/checks.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import * as vscode from 'vscode';
+
+export function extensionsChecks() {
+	const extensionHandlers = {
+		"sergey-agadzhanov.AMPscript": {
+			handler: () => {
+				vscode.window.showInformationMessage(`MCFS [AMPScript] extension is currently not supported by AMPscript Core.`);
```

Comment:
Let's update this message to convey what not supported means.

"The extension MCFS [AMPScript] is incompatible with AMPscript Core and will prevent AMPscript Core from debugging AMPscript."